aten/src/ATen/native/TensorFactories.cpp

```diff
@@ -595,6 +595,22 @@ Tensor hann_window(
       window_length, periodic, /*alpha=*/0.5, /*beta=*/0.5, options);
 }
 
+// ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~ conversions ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Tensor cpu(const Tensor& self) {
+  return self.to(kCPU);
+}
+
+Tensor cuda(const Tensor& self, int64_t device_id, bool non_blocking) {
+  if (device_id == -1) { // default argument
```

Comment:
This is not necessary, `Device` accepts `-1` as the default index (current device). This line `return self.to(kCUDA, non_blocking);` is actually equivalent to `return self.to({kCUDA, -1}, non_blocking);`, so the whole function should just be `return self.to({kCUDA, device_id}, non_blocking);`.